lib/Conf.ml

```diff
@@ -215,16 +215,16 @@ module Formatting = struct
     let doc = "Position of the assignment operator." in
     let names = ["assignment-operator"] in
     let all =
-      [ ( "end-line"
-        , `End_line
-        , "$(b,end-line) positions assignment operators (`:=` and `<-`) at \
-           the end of the line and breaks after it if the whole assignment \
-           expression does not fit on a single line." )
-      ; ( "begin-line"
-        , `Begin_line
-        , "$(b,begin-line) positions assignment operators (`:=` and `<-`) \
-           at the beginning of the line and breaks before it if the whole \
-           assignment expression does not fit on a single line." ) ]
+      [ C.Value.Ok.valid ~name:"end-line" `End_line
+          ~doc:
+            "$(b,end-line) positions assignment operators (`:=` and `<-`) \
+             at the end of the line and breaks after it if the whole \
+             assignment expression does not fit on a single line."
+      ; C.Value.Ok.valid ~name:"begin-line" `Begin_line
+          ~doc:
```

Comment:
I think the labels are not necessary. Removing them might reduce the diff.

Reply:
done